src/SumOfThree.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Created by Yura on 01.05.2019.
+ */
+public class SumOfThree {
+    public static void main(String[] args){
+
+        int n=0;
+
+        for(int i=0;i<1000000;i++){
+            int i1=i/100000%10,
```

Comment:
Too many redundant variables. 2 are enough. You can even avoid using them at all.